sylt-parser/src/expression.rs

```diff
@@ -798,6 +803,9 @@ mod test {
     test!(expression, fn_invalid_empty_tuple: "fn -> () end" => _);
 
     fail!(expression, fn_self_arg: "fn self: int do 1 end" => _);
+    fail!(expression, list_funky: "[1 2]" => _);
+    fail!(expression, set_funky: "{1 2}" => _);
+    fail!(expression, dict_funky: "{1: 2 3: 4}" => _);
```

Comment:
```suggestion
    fail!(expression, dict_funky: "{1: 2 3: 4}" => _);
    fail!(expression, tuple_funky: "(1 2 3 4}" => _);
```
Add tuples for completeness.